tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/PackageRestore/Snapshots/RestoreBuilderTests.cs

```diff
@@ -58,9 +57,9 @@ public void ToProjectRestoreInfo_WhenNuGetRestoreRuleMissing_ReturnsEmpty()
 
             Assert.Empty(result.MSBuildProjectExtensionsPath);
             Assert.Empty(result.OriginalTargetFrameworks);
-            Assert.Equal(1, result.TargetFrameworks.Count);
+            Assert.Single(result.TargetFrameworks);
 
-            var targetFramework = result.TargetFrameworks.Item(0);
+            var targetFramework = result.TargetFrameworks[0];
```

Comment:
`Single` returns the item, so you can just write:

```c#
var targetFramework = Assert.Single(result.TargetFrameworks);
```

Reply:
TIL 👍🏻

Reply:
Done.